homeassistant/components/template/config.py

```diff
@@ -163,10 +155,12 @@ async def _async_resolve_blueprints(
         raw_config = dict(config)
 
     if is_blueprint_instance_config(config):
-        config = TEMPLATE_BLUEPRINT_INSTANCE_SCHEMA(config)
         blueprints = async_get_blueprints(hass)
 
         blueprint_inputs = await blueprints.async_inputs_from_config(config)
+        blueprint_inputs = await blueprints.async_inputs_from_config(
```

Comment:
This looks weird. Why do we use a variable if we overwrite it before using it?

Reply:
Oops, that was a mistake. I've removed the duplicated line.